clj2nix.clj

```diff
@@ -63,16 +64,17 @@ let repos = [" (repos-nix mvn-repos) " ];
           ""
           (format "classifier = \"%s\";" classifier))]
    (format "
-  {
+  rec {
     name = \"%s\";
-    paths = [(fetchmaven {
+    src = fetchmaven {
       inherit repos;
       artifactId = \"%s\";
       groupId = \"%s\";
       sha512 = \"%s\";
       version = \"%s\";
       %s
-    })];
+    };
+    paths = [(src)];
```

Comment:
The parenthesis around `src` is unnecessary here.

Reply:
oh derp, yeah... I know better than that...